packages/seacas/libraries/ioss/src/Ioss_Utils.C

```diff
@@ -2006,6 +2049,605 @@ void Ioss::Utils::copy_database(Ioss::Region &region, Ioss::Region &output_regio
   }
 }
 
+bool Ioss::Utils::compare_database(Ioss::Region &input_region, Ioss::Region &output_region,
```

Comment:
What is this needed for?

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.